.github/workflows/performance.yml

```
@@ -95,3 +95,11 @@ jobs:
                   name: failures-artifacts
                   path: ./__test-results/artifacts
                   if-no-files-found: ignore
+
+            - name: Run Reassure performance tests
+              run: cd gutenberg/bin && ./reassure-tests.sh
+
+            - name: Run Danger.js
```

Similar to my comment on Reassure, I think Danger.js is an implementation detail that may provide little value to the observer of this CI task. Maybe we can identify a description that better communicates intent? 

```suggestion
            - name: Report mobile performance results
```